r2i/tflite/prediction.h

```diff
@@ -24,14 +24,16 @@ class Prediction: public IPrediction {
  public:
   Prediction ();
   ~Prediction ();
-  double At (unsigned int index,  r2i::RuntimeError &error) override;
-  void *GetResultData () override;
-  unsigned int GetResultSize () override;
-  RuntimeError SetTensorValues(float *outputdata, int tensorsize);
+  double At (unsigned int output_index, unsigned int index,
+             r2i::RuntimeError &error) override;
+  void *GetResultData (unsigned int output_index, RuntimeError &error) override;
+  unsigned int GetResultSize (unsigned int output_index,
+                              RuntimeError &error) override;
+  RuntimeError AddResults (float *data, unsigned int size) override;
 
  private:
-  float *outputdata = NULL;
-  int tensorsize;
+  std::vector<std::shared_ptr<float>> results_data;
+  std::vector<unsigned int> results_sizes;
```

Comment:
Why instead of having two vectors, one for the data and one for the resultads, we just use vector that stores tuples (data, size)? This is because we are assuming that entry 0 in  results_data will correspond to entry 0 in results_sizes, I believe this could be dangerous.

Reply:
This file has been removed.